public/js/pimcore/object/classes/data/multiselect.js

```diff
@@ -256,84 +394,128 @@ pimcore.object.classes.data.multiselect = Class.create(pimcore.object.classes.da
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('up'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/up.svg",
+                            handler: function (grid, rowIndex) {
+                                if (rowIndex > 0) {
+                                    var rec = grid.getStore().getAt(rowIndex);
+                                    grid.getStore().removeAt(rowIndex);
+                                    grid.getStore().insert(--rowIndex, [rec]);
+                                    model.select(rowIndex);
+                                }
+                            }.bind(this)
+                        }
+                    ]
+                },
+                {
+                    xtype: 'actioncolumn',
+                    menuText: t('down'),
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('down'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/down.svg",
+                            handler: function (grid, rowIndex) {
+                                if (rowIndex < (grid.getStore().getCount() - 1)) {
+                                    const rec = grid.getStore().getAt(rowIndex);
+                                    grid.getStore().removeAt(rowIndex);
+                                    grid.getStore().insert(++rowIndex, [rec]);
+                                    this.defaultValueModel.select(rowIndex);
+                                }
+                            }.bind(this)
+                        }
+                    ]
+                },
+                {
+                    xtype: 'actioncolumn',
+                    menuText: t('remove'),
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('remove'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/delete.svg",
+                            handler: function (grid, rowIndex) {
+                                grid.getStore().removeAt(rowIndex);
+                            }.bind(this)
+                        }
+                    ]
+                }
+            ],
+            autoHeight: true,
+            plugins: [
+                Ext.create('Ext.grid.plugin.CellEditing', {
+                    clicksToEdit: 1,
+                    listeners: {
+                        edit: function(editor, e) {
+                            if(!e.record.get('value')) {
+                                e.record.set('value');
+                            }
+                        },
+                    }
+                })
+            ]
+        });
     },
-
-    showoptioneditor: function (valueStore) {
-        var editor = new pimcore.object.helpers.optionEditor(valueStore);
-        editor.edit();
```

Comment:
might need some tweak because we only have one column in `Default Value`
![image](https://github.com/user-attachments/assets/10b3ad1c-630c-48a0-a935-f332eec8de56)
opposed to `Selection Options`
![image](https://github.com/user-attachments/assets/73735669-2335-44d7-9cd7-50425d4fa096) and the values are all starting with an `,something`